devito/types/dense.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 import sympy
+import warnings
```

Comment:
if it's from the standard lib, it belongs to the first group of imports (if... I don't remember)